.snyk

```diff
@@ -0,0 +1,7 @@
+# References:
+# https://docs.snyk.io/scan-applications/snyk-code/using-snyk-code-from-the-cli/excluding-directories-and-files-from-the-snyk-code-cli-test
+# https://docs.snyk.io/snyk-cli/commands/ignore
+exclude:
+  global:
+    - vendor/**
```

Comment:
Is it better to scan through the projects vendor for vulnerabilities unless it is made sure the dependent packages are scanned independently.